core/common/assets/js/api/modules/errors/internal-server-error.js

```diff
@@ -0,0 +1,9 @@
+import BaseError from './base-error';
```

Comment:
Why do you need this error if it doesn't change anything?

Reply:
This was an example of a normalize error that returns from the server.
for this PR this is not required, I will remove it.